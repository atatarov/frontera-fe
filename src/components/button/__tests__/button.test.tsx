import React from 'react';
import { mount } from 'enzyme';
import { describe, it, expect } from '@jest/globals';
import  Button from '../';


function sum(a, b) {
    return a+b;
}
const array = [1,2,3,4,5];

describe('Тестируем компонент button', () => {
    it('Тестируем рендер компонента button с текстом', ()=> {
        const ButtonTest = mount(<Button> Test 2 </Button>)
        expect(ButtonTest).toMatchSnapshot();
    })
    it.each(array)('Тестируем sum', (item)=> {
        expect(sum(item,2)).toBe(3);
    })
})  
