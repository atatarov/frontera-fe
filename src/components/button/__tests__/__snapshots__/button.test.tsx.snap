// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестируем компонент button Тестируем рендер компонента button с текстом 1`] = `
<Button>
  <button
    className="button"
  >
     Test 2 
  </button>
</Button>
`;
